pages/playerlist/index.js

```diff
@@ -0,0 +1,22 @@
+import useSWR from "swr";
+
+export default function PlayerList() {
+  const { data = [] } = useSWR("/api/players");
+  console.log(data);
+  return (
+    <>
+      <h1>Hello Players:</h1>
+      <ul>
+        {data.map((player) => {
+          return (
+            <>
+              <li key={player._id}>
+                Name: {player.name} E-Mail:{player.email}
+              </li>
+            </>
```

Comment:
You don't need the `fragment` (`<></>`) here. Just use the `li` element. Don't forget the `key` prop.

Reply:
alright, i deleted the fragment. The page itself is something for a different US. 
The key prop is already on the list item?! Or what do you mean?